src/main/java/pl/senla/hotel/dto/HotelServiceCreateDto.java

```diff
@@ -0,0 +1,32 @@
+package pl.senla.hotel.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+@Data
+@Builder
+public class HotelServiceCreateDto implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 120L;
+
+    private Integer idOrder;
+    private Integer idGuest;
+    private Integer idHotelFacility;
+    private Integer numberOfDays;
+    private String checkInTimeString;
```

Comment:
you can provide it as a Local/Zoned dateTime or just Date depends on what you used in entity